### ISSUE TITLE
Trying to build Jinja2Cpp with latest version of CMake - Have errors about Nlohmann::JSON and RapidJSON

### ISSUE THREAD

**Issue by DragonOsman** [jinja2cpp/Jinja2Cpp#202]
I tried to generate project files for Jinja2Cpp using the latest version of CMake, 3.18.1.  I tried with both clang-cl.exe and cl.exe as the compiler for VS2019.  In both cases, I got this CMakeError:
```
-- RapidJSON found. Headers: C:/Jinja2Cpp/thirdparty/json/rapid/include
fatal: Needed a single revision
Unable to find current revision in submodule path 'thirdparty/json/nlohmann'
CMake Error at thirdparty/internal_deps.cmake:41 (add_subdirectory):
  The source directory

    C:/Jinja2Cpp/thirdparty/json/nlohmann

  does not contain a CMakeLists.txt file.
Call Stack (most recent call first):
  thirdparty/thirdparty-external-boost.cmake:3 (include)
  thirdparty/CMakeLists.txt:44 (include)
  CMakeLists.txt:172 (include)
```
Please help me out here if possible.  Is there a way to resolve this?

**Comment by satarsa** [comment]
I have the same issues with both VC2019 and VC2017 after their upgrade up to 16.7.1 and 15.9.26 respectively... Unfortunately, I guess, it is not going to be fixed...

**Comment by DragonOsman** [comment]
I'm also having the problem after upgrade to VS2019 version 16.7.1.

Should we mention it on MSDN Forums?  I have the problem on both Developer Command Prompt for VS2019 and also in the VS2019 IDE with its CMake support.

**Comment by satarsa** [comment]
May be this pull request would help? 
https://github.com/jinja2cpp/Jinja2Cpp/pull/197
Could you try? I'm far from work for now, I'll be able to test it only after a few days.

**Comment by DragonOsman** [comment]
I'm not using a Conan package, though.  I'm trying to build it directly.  So I don't think that will help me.  Thanks, though.

**Comment by DragonOsman** [comment]
I was able to solve that by re-cloning the library and then updating submodules.  

But now I have a problem where I get build errors (I managed to successfully generate project files).  I'll open a new issue for that.
